src/components/Toolbar/index.jsx

```diff
@@ -0,0 +1,121 @@
+/* global cozy */
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Menu, { Item } from 'components/Menu'
+import styles from './index.styl'
+
+import { withBreakpoints } from 'cozy-ui/react'
+
+const { BarRight } = cozy.bar
+
+class Toolbar extends Component {
+  render() {
+    const {
+      children,
+      moreMenuTitle,
+      moreMenuDisabled,
+      moreMenuClassName,
+      moreMenuButton,
+      breakpoints: { isMobile }
+    } = this.props
+
+    const actions = React.Children.toArray(children)
+    const highlightedActions = actions.filter(child => child.props.highlighted)
+    const secondaryActions = actions.filter(child => !child.props.highlighted)
+
+    const MoreMenu = (
+      <Menu
+        title={moreMenuTitle}
+        disabled={moreMenuDisabled}
+        className={moreMenuClassName}
+        button={moreMenuButton}
+      >
+        {highlightedActions.map(component =>
+          this.wrapInItem(
+            React.cloneElement(component, {
+              highlighted: false,
+              isSecondaryAction: false
+            })
```

Comment:
Because `Component` is not actually a component but a component instance — so I should rename it.
Trying what you say gives an error like `Unable to createElement because tag name is [Object object]`, because `Component` is a VNode at this point. *Maybe* `<Component.type />` would work but is that really better?

Reply:
```js
const Component = props => <div>Hello</div>
const ComponentInstance = <div>Hello</div>
```

JSX covers both, doesn't it?